recipes/howardhinnant_date/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "howardhinnant_date" %}
+{% set version = "2.4.1" %}
+{% set sha256 = "98907d243397483bd7ad889bf6c66746db0d7d2a39cc9aacc041834c40b65b98" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/HowardHinnant/date/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:  # Empty host dependency section
+    - libcurl
+  run:
+    - libcurl
```

Comment:
If this doesn't rely on libcurl, you can remove these lines

Reply:
Thanks for reviewing!
Unless `USE_SYSTEM_TZ_DB=ON` is passed to `cmake` cmd line, `libtz.so` does depend on `libcurl`.